src/components/NavBar.tsx

```diff
@@ -0,0 +1,32 @@
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const NavBar: React.FC = () => (
+  <nav
+    className="navbar is-light is-fixed-top is-mobile has-shadow"
+    data-cy="Nav"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink
+          to="/"
+          className={({ isActive }) => classNames(
+            'navbar-item',
+            { 'is-active': isActive },
+          )}
+        >
+          Home
+        </NavLink>
+        <NavLink
+          to="/tabs"
+          className={({ isActive }) => classNames(
+            'navbar-item',
+            { 'is-active': isActive },
+          )}
+        >
+          Tabs
+        </NavLink>
```

Comment:
We can create our custom component to avoid code duplication. You can rewatch [this video](https://mate.academy/learn/react/react-router?section=video&videoId=3348)